javascript/javascript3/week2/promise.js

```diff
@@ -0,0 +1,14 @@
+//task2 Promise that resolves after set time
+function resolveAfterSetTime(resolveAfter) {
+    return new Promise((resolve) => {
+      setTimeout(() => {
+        resolve();
```

Comment:
Expected 1 arguments, but got 0. Did you forget to include 'void' in your type argument to 'Promise'?ts(2794)

Reply:
`resolve` expects an argument. For example `resolve('timeout callback called`)`